server/cmake/modules/FindLibClang.cmake

```diff
@@ -61,20 +61,20 @@ endforeach()
 
 find_path(LIBCLANG_INCLUDE_DIR clang-c/Index.h
   PATHS ${libclang_llvm_header_search_paths}
-  PATH_SUFFIXES LLVM/include #Windows package from http://llvm.org/releases/
+  PATH_SUFFIXES llvm/include #Windows package from http://llvm.org/releases/
```

Comment:
Irony was still able to build (on OS X) with upper-case `LLVM`, although on my OS X system the path was indeed `llvm`.